Disable tqdm progressbars if no TTY is present

mvanbaak opened the pull request:
When running without TTY (FreeBSD cron for example) tqdm tries to paint the progress bars, resulting in a non clean exit, and thus TCM does not complete.
By disabling the tqdm progress bar when not attached to a TTY, it does complete.

There will be 0 change for when TCM is run from the CLI.

CollinHeist commented:
Thanks for the PR, this is a great idea. If you could submit this to the `develop` branch I'll get this merged and then added to the next release.

Thanks!

mvanbaak commented:
> If you could submit this to the `develop` branch I'll get this merged and then added to the next release.

Rebased my branch on top of `develop` and changed the base for this PR.
Thanks for accepting the fix

CollinHeist reviewed (approved):
Perfect. Thanks again!
